src/handlers.ts

```diff
@@ -184,14 +184,8 @@ EXPECTED RESULT
           exist per test target at a time.`),
     },
     async (params, {sessionId}) => {
-      if (!sessionId) {
```

Comment:
Should we have a test this now works even if the sessionId is empty? 🤷

Reply:
Yeah, probably? Then for the other handlers as well?